project-api/project-api-test/src/test/resources/projects_for_resolution_integration_tests/package_aa_1_0_0/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org = "adv_res"
+name = "package_aa"
+version = "1.0.0"
+export = ["package_aa"]
```

Comment:
Can we leave last line as a empty line in these files?

Reply:
Added lasntline whereever applicable